index.js

```diff
@@ -75,8 +75,10 @@ function compilerErrorToString(err, pathToElm) {
       default:
         return ("Error attempting to run Elm compiler \"" + pathToElm + "\":\n" + err);
     }
+  } else if ((typeof err === "object") && (typeof err.message === "string")) {
+    return (JSON.stringify(err.message));
```

Comment:
no need for `()` around `JSON.stringify(err.message)` here

Reply:
Fixed